gatsby-config.js

```diff
@@ -59,10 +60,23 @@ module.exports = {
             refetch: Boolean(process.env.BUILD_RELATED_CONTENT),
             engineKey: 'Ad9HfGjDw4GRkcmJjUut',
             limit: 5,
-            getSlug: ({ node }) => node.frontmatter.path,
+            getSlug: ({ node }) => {
+              if (node.internal.type === 'Mdx') {
+                return node.frontmatter.path;
+              } else if (node.internal.type === 'Quickstarts') {
```

Comment:
was an `else if` used instead of just an `else` because we might be adding more `node.internal.type`s eventually?

Reply:
mostly just to be extra explicit for the heck of it 😂